src/open-hackathon-server/Kaiyuanshe.OpenHackathon.Server/Storage/BlobContainers/BlobContainerNames.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Kaiyuanshe.OpenHackathon.Server.Storage.BlobContainers
+{
+    public static class BlobContainerNames
+    {
+        public static readonly string User = "kysuser";
```

Comment:
It's not for `user` only. Let's make it a general term

Reply:
The underlying logic is the same as what is described here: https://github.com/kaiyuanshe/open-hackathon/pull/1013/files#r659162807 

let's make the design consistent.